utils/chainMap.ts

```diff
@@ -0,0 +1,25 @@
+const chainMappings = {
+  // Chain name → URL slug
+  Ethereum: 'ethereum',
```

Comment:
i think these maps should be ID based rather than name based. You can get name from the chain objects (from viem/wagmi) themselves.